packages/react/src/components/ComposedModal/next/ComposedModal.stories.js

```diff
@@ -296,10 +298,19 @@ export const WithStateManager = () => {
   return (
     <ModalStateManager
       renderLauncher={({ setOpen }) => (
-        <Button onClick={() => setOpen(true)}>Launch composed modal</Button>
+        <Button ref={closeButton} onClick={() => setOpen(true)}>
+          Launch composed modal
+        </Button>
       )}>
       {({ open, setOpen }) => (
-        <ComposedModal open={open} onClose={() => setOpen(false)}>
+        <ComposedModal
+          open={open}
+          onClose={() => {
+            setOpen(false);
+            setTimeout(() => {
+              closeButton.current.focus();
+            });
```

Comment:
@joshblack yeah I found it only worked when I used the `setTimout()`, otherwise the focus kept getting stuck on the CTA within the no longer visible modal